Modules/DevTools/BusinessCentralPerformanceToolkit/src/BCPTLogEntry.Table.al

```diff
@@ -5,6 +5,7 @@
 table 149002 "BCPT Log Entry"
 {
     DataClassification = SystemMetadata;
+    DrillDownPageId = "BCPT Log Entries";
```

Comment:
Is this change related?

Reply:
No, sorry, this should not be in the PR. See #9719. As described there I wanted to change another PR for that, but GitHub seems not to allow me. Or even worse: it incorporates it in the existing PR. Any tips on how to separate PRs are more than welcome.

Reply:
Hi @lvanvugt you should look into branches. I responded to your issue.